README.md

```diff
@@ -1,6 +1,6 @@
 # AWS Deposit Service

```

Comment:
Lets change this from AWS Deposit Service to something more generic if we are going to mention GCS here too. Otherwise we could make a separate section. It would probably be good to mention required permissions for a GCS service account that would be required to get this to work. You could just mention the same permissions you tested with.

Reply:
Renamed from AWS Deposit Service to just Message Deposit Service and added a note under the Quick Run section that includes the required permissions for the GCP service account